js/models/questionModel.js

```diff
@@ -65,6 +65,7 @@ class QuestionModel extends ComponentModel {
   }
 
   init() {
+    this._contextActivities = [];
```

Comment:
Change throughout to "activity context"? Are we adding activity context to the questions? Or are we adding context activities to the questions?